### PR TITLE
Add webpack

### PR DESCRIPTION
**Pull request by Victoria129**
I have added webpack to the project as well as linter files.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello World!");
```
Comment (girma336) [inline src/index.js:1]: - You should remove all console.log For best practice

**Review by girma336 — APPROVED** [review]
- nothing to say other than great job

File: src/index.js (hunk 1 above)
Comment (Victoria129) [reply src/index.js]: Yes @girma336, sure. Thank you. I have made the necessary changes.
Kindly review again.

**Review by girma336 — APPROVED** [review]
- well done

**Pull request merged**